providers/base/units/tracers/jobs.pxu

```diff
@@ -0,0 +1,15 @@
+plugin: shell
+id: tracers/ptrace
+command: ptrace_test
+_summary:
+ Tests the ptrace tracer availability
+_description:
+ Runs a test for ptrace tracer availability.
```

Comment:
Similarly to the one below, the description could be more specific. For instance:
The test initiates tracing using the `ptrace` syscall from the c code.